app/src/main/java/com/framgia/fsalon/screen/bill/BillViewModel.java

```diff
@@ -4,6 +4,10 @@
 import android.content.Context;
 import android.databinding.BaseObservable;
 import android.databinding.Bindable;
+import android.os.Parcel;
+import android.support.v4.app.DialogFragment;
```

Comment:
ERROR: Unused import: android.support.v4.app.DialogFragment. Use Ctrl+Shift+O (Eclipse) or Ctrl+Alt+O (Intellij) to sort imports